firebase.json

```diff
@@ -45,7 +45,7 @@
           {
             "key": "Content-Security-Policy",
             // Keep in sync with `angular.json`.
```

Comment:
I'm not sure what this comment means, I don't see this being set in angular.json

Reply:
It's this one: https://github.com/angular/material.angular.io/blob/main/angular.json#L114

Reply:
Huh, don't know why I couldn't find that